python/ola/UIDTest.py

```diff
@@ -109,6 +110,7 @@ def testCmp(self):
     self.assertFalse(u3 == u2)
 
     self.assertNotEqual(u3, u4)
+    self.assertNotEqual(hash(u3), hash(u4))
```

Comment:
Can we also test hash comparison of u2 and u3, to ensure the manufacturer ID is in the hash correctly.

Reply:
that's actually handled by allHashNotEqual below

Reply:
Ah yes, thanks.